scripts/samples/bench_and_plot.py

```diff
@@ -339,13 +339,12 @@ def bench(args: Namespace, bench_name: str, desc: str, generate: Callable,
     plt.ylabel('Probability')
     plt.grid(which='major', ls='-', c='#ddd', lw=0.5)
     plt.grid(which='minor', ls=':', c='#ddd', lw=0.5)
+    formatter = ScalarFormatter()
```

Comment:
Pyright was complaining `error: Cannot access member "set_scientific" for type "Formatter"`. Hence, this change.